src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,61 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * This class encapsulates the UI logic for the application.
+ */
+public class Ui {
+    private static final String LOGO =
+            " ____        _        \n"
+                    + "|  _ \\ _   _| | _____ \n"
+                    + "| | | | | | | |/ / _ \\\n"
+                    + "| |_| | |_| |   <  __/\n"
+                    + "|____/ \\__,_|_|\\_\\___|\n";
+    private static final String LINE = "-----------------------------------------------------------------------\n";
+    private final Scanner sc;
```

Comment:
Perhaps a more obvious name for the scanner would be better? Though I agree this should be intuitive enough.